plugins/amdgpu/fu-amdgpu-device.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2023 Advanced Micro Devices Inc.
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#include "fu-amdgpu-device.h"
```

Comment:
i think the plugin should be amd-gpu and have the source file prefix of fu-amd-gpu TBH.

Reply:
Even though the kernel driver is 'amdgpu'?

Reply:
I think so; I think we're settling on `vendor-thing` as plugin names -- for instance the Intel-GSC plugin is `igsc` in the kernel.

Reply:
OK.